scripts/yocto-build-and-test.sh

```diff
@@ -29,6 +29,14 @@ is_testing_board() {
     return $ret
 }
 
+bitbake_override_separator() {
+    if fgrep -q ':${PN}' "$WORKSPACE/meta/conf/bitbake.conf"; then
+        echo -n ":"
+    else
+        echo -n "_"
+    fi
```

Comment:
@TheYoctoJester: Can you think of a better/more standardized way of determining this in the shell?

Reply:
@kacf The failsafe way would be to check for `LOCALCONF_VERSION`. 1 is the old, `_` style, 2 is the new `:` style. See also: https://git.yoctoproject.org/poky/tree/meta/conf/sanity.conf#n11

Reply:
Thanks, I'll use that.